src/dotnet/APIView/APIView/CodeFileRenderer.cs

```diff
@@ -24,23 +25,40 @@ private void Render(List<CodeLine> list, IEnumerable<CodeFileToken> node)
             string currentId = null;
             bool isDocumentation = false;
             bool isDeprecatedToken = false;
+            //This will be set to true by default when a new line starts and 
+            // set to false when any non documentation token is found within the line
+            bool isLineAllDocumentation = true;
 
             foreach (var token in node)
             {
                 switch(token.Kind)
                 {
                     case CodeFileTokenKind.Newline:
-                        list.Add(new CodeLine(stringBuilder.ToString(), currentId));
+                        //Close documentation span if within doc range
+                        if(isDocumentation)
```

Comment:
Formatting, here and elsewhere.
```suggestion
                        if (isDocumentation)
```